mysql-test/suite/rocksdb/t/ttl_primary_with_partitions.test

```diff
@@ -0,0 +1,245 @@
+--source include/have_rocksdb.inc
```

Comment:
Doesn't exist in upstream, I'll just add it to ours on the commit introducing the test and put it into the great list of upstream mtr diffs.